src/types/document.rs

```diff
@@ -29,7 +29,7 @@ pub struct Document {
     pub file_name: Option<String>,
 
     /// A MIME type of the file as defined by a sender.
-    #[serde(with = "crate::types::non_telegram_types::mime::opt_deser")]
+    #[serde(default, with = "crate::types::non_telegram_types::mime::opt_deser")]
     pub mime_type: Option<Mime>,
```

Comment:
I wander if we have the same problems with dates somewhere?

Reply:
Likely. I'll search for these cases.

Reply:
I hope they all match `:[ ]+DateTime`, because I am lazy!

Reply:
You can also try searching for `serde(with` or something like that

Reply:
`:[ ]+Option<DateTime`, I meant. Btw, why does this exist? https://github.com/teloxide/teloxide-core/blob/master/src/types.rs#L248-L250

Reply:
Because at that time I forgot that `#[serde(default)]` exists and thought that I need a function to do `#[serde(default = "none")]`

Reply:
Oh ok